disperser/apiserver/rate_config.go

```diff
@@ -10,12 +13,18 @@ const (
 	RegisteredQuorumFlagName        = "auth.registered-quorum"
 	TotalUnauthThroughputFlagName   = "auth.total-unauth-throughput"
```

Comment:
Should we call these "byte rate", to make it clear compared to "blob rate"?

Reply:
I don't have a strong opinion. Keep in mind the units are provided in the help menu. I'll change it if you prefer `byte-rate`